cookbook/projects/policy_eth_with_addon.yml

```diff
@@ -0,0 +1,9 @@
+Policy:
+  selected_providers_mode: ALLOWED
+  chain_policies:
```

Comment:
you disabled all other chains, that needs to be fixed

Reply:
I thought that this is the purpose of `selected_providers_mode`.

![image](https://github.com/lavanet/lava/assets/6321801/a3fcc699-3880-46b7-8b50-12f592d94df4)

Reply:
it's not, that flag is on how we select providers not how we define the chains we can use, look at the policy we are using it has a wildcard